### ISSUE TITLE
I'm able to open some of these up in my browser

### ISSUE THREAD

**Issue by jawz101** [dead-hosts/adaway.github.io_git_AdAway#1]
@funilrys 

https://github.com/dead-hosts/adaway.github.io_git_AdAway/blob/cba24c5349b0436f5739a56c23bdd9fc19c72280/output/hosts/INACTIVE/hosts

I just tried a few and these open up in a web browser for me just fine

```
vslider.admedia.com
injection.admedia.com
m.admedia.com
```

Can the checks do something to re-process the entries marked as INACTIVE prior to outputting them as such?  Maybe a cooldown period to see if the whois servers or your ISP aren't flagging the lookups as a denial of service?  I know when I was running my list thru PyFunceble my ISP killed my connection once and I got a notice to check my computers for malware because they detected the aggressive lookups and started blocking my connections.

tldr I've had my ISP detect anomalous traffic on my connection.  I've also read whois server Terms of Services that say they will curb aggressive whois lookups.
